### DISCUSSION TITLE
fetch items that only have a specific property

### DISCUSSION THREAD

**Discussion by nabildroid** [deta/cloud-docs#462] [Help]
for example, all items that have expire 

[
{key:"k1",expire:"111"},
{key:"k2"}
]

how to write a query that returns {key:"k1",expire:"111"},

**Comment by abdelhai** [comment]
`fetch({"expire": "111"})`

**Comment by jnsougata** [comment]
There's currently no way to check if a `field` exists or not iirc. One thing you can do is use the [Greater Than or Equal](https://docs.deta.sh/docs/base/queries#greater-than-or-equal) like this:  `{"expire?gte": 0}`  ......might be a good work around if the value for `expire` is not constant

**Reply by nabildroid** [reply]
so, we can't check field is not empty?

**Reply by jnsougata** [reply]
it will automatically overlook those records which doesn't have `expire` field

**Reply by nabildroid** [reply]
ok thank you <3
